README.md

```diff
@@ -800,3 +801,155 @@ global def pio16DUT =
+Specifically, add the following line to `demo.wake`. 
+```wake
+publish scribbleDirectories = simplify "../../docs/scribble", Nil
+```
+
+### Creating an Onboarding document.
+At this point, we can create the first draft of the PIO Onboarding document. Type the following command.
+```
+wake makeOnboardingDocument pioDUT
+```
+The Onboarding document is a short test document which displays all the sections we've written so far. 
+It is created in the directory `build/api-generator-sifive/pioDUT/documentation`. 
+
+Note we haven't actually written anything about PIO yet. 
+The Onboarding document includes placeholder sections which fill in the missing PIO sections.
+Consequently, we can create an Onboarding document at any time, even if we haven't written the documentation yet.
+
+The next step replaces those placeholders with text specific to the Parallel I/O block. 
+We will create three Jinja templates `Overview.jinja2`, `Programming.jinja2` and `HardwareInterface.jinja2`, 
+all of which reside in the `docs/scribble/components/pio` directory.
+
+#### Create the Overview template.
+The Overview template contains a simple paragraph describing what the block does. 
+While the pioDUT configuration has only a single instance of PIO,
+the paragraph should be crafted to allow for multiple instances.
+
+To get started, copy the following text into the file `Overview.jinja2`.
+```
+= Parallel I/O (PIO)
+
+The {{ product_name }} contains {{ devices|length|english_number }} parallel I/O (PIO) block{{ devices|pluralize }} for simple parallel input/output.
+The PIO block is further described in <<chapter-pio>>.
+```
+The functions and variables used in this template are described in [Scribble Test Socket](https://github.com/sifive/scribble-testsocket-sifive). 
+This is a good time rerun the wake command and update the document. (`wake makeOnboardingDocument pioDUT`)
+You should see an introductory paragraph:
+```
+The pioDUT Test Socket contains one parallel I/O (PIO) block for simple parallel input/output.
+The PIO block is further described in Parallel I/O (PIO).
+```
+
+#### Create the Programming template.
+The Programming chapter goes into more detail about what the block does and how to use it. 
+For PIO, it should give a general description of a Parallel I/O device and provide information on each instance in the design.
+If the block has registers, this chapter should invoke `RegisterMap` 
+to display a register map along with descriptions of all the register fields.
+(See [Scribble Test Socket](https://github.com/sifive/scribble-testsocket-sifive) for more information on `RegisterMap`.)
+
+For pioDUT, we are going to create the main `Programming` section which invokes `RegisterMap`, 
+and we're going to create a subsection called `InstanceTable` which displays the configurable values of each PIO instance.
+
+First, copy the following text into the main section `Programming.jinja2`.
+```
+{%  set registers = RegisterMap(scope) %}
```

Comment:
```suggestion
{% set registers = RegisterMap(scope) %}
```

Sorry to nitpick about this, but let's remove the second space character before the word `set` here. I'll remove it from the actual Jinja template file in the `master` branch.

Reply:
Thanks for proofreading! It really helps.